src/config/AxiosInstance.ts

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const AxiosInstance = axios.create({
+  baseURL: 'http://localhost:3000',
```

Comment:
@kleyla This information should not be hardcoded, at least can be moved to variables or better to environment variables.

Reply:
@Carlos-YD  I have fixed that and deleted the library that appears in alert from security of sonarcloud, thank you for the review :)